thirdparty/vars.sh

```diff
@@ -410,10 +410,10 @@ LLVM_SOURCE="llvm-project-16.0.6.src"
 LLVM_MD5SUM="dc13938a604f70379d3b38d09031de98"
 
 #clucene
-CLUCENE_DOWNLOAD="https://github.com/StarRocks/clucene/archive/refs/tags/starrocks-2023.11.06.tar.gz"
-CLUCENE_NAME="starrocks-clucene-2023.11.06.tar.gz"
-CLUCENE_SOURCE="starrocks-clucene-2023.11.06"
-CLUCENE_MD5SUM="6837d64c30c2559c6c512e279856776f"
+CLUCENE_DOWNLOAD="https://github.com/StarRocks/clucene/archive/refs/tags/starrocks-2023.11.10.tar.gz"
```

Comment:
Please make all thirdparty changes in a separate pull request as far as you can.

Reply:
@imay fix at #34822.